charts/kueue/values.yaml

```
@@ -137,7 +137,7 @@ metricsService:
     - name: https
```

```suggestion
    - name: metrics
```

Let's say `metrics` as well.